Assets/MixedRealityToolkit/_Core/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -27,10 +27,15 @@ public class MixedRealityBoundaryManager : BaseManager, IMixedRealityBoundarySys
         public bool EnablePlatformBoundaryRendering { get; set; } = true;
```

Comment:
Getting some warnings about `Scale`, `BoundaryHeight`, and `EnablePlatformBoundaryRendering` are all set in the constructor and their values are ignored during initialization.

Reply:
that's odd... let me investigate

Reply:
where are you seeing these? i dont see them in the unity console window or in VS

Reply:
and they are definitely being referenced in the implementation

Reply:
The values are overwritten by the constructor. So it's redundant to set them in the property initializer.